### PR TITLE
Add analyses schemas

### PR DESCRIPTION
**Pull request by nagem**
Checks input schemas for external and job-based analyses.

### CODE REVIEW

File: api/placer.py [hunk 1]
```diff
@@ -506,7 +506,7 @@ class AnalysisPlacer(Placer):
 
     def check(self):
         self.requireMetadata()
-        #validators.validate_data(self.metadata, 'analysis.json', 'input', 'POST', optional=True)
+        validators.validate_data(self.metadata, 'analysis.json', 'input', 'POST', optional=True)
```
Comment (coltonlw) [inline api/placer.py]: 👍

**Comment by coltonlw** [comment]
@nagem Since you spent the extra few minutes to add a jsonschema for proper input validation, I returned the favor wrote a postman test to upload a file to /engine with level=analysis and tried it in a branch merged with your changes.  The test passed so from that aspect this LGTM!

https://travis-ci.org/scitran/core/builds/153542448#L2525

https://github.com/scitran/core/blob/test-placer/test/integration_tests/postman/integration_tests.postman_collection#L14

**Comment by ryansanford** [comment]
@nagem 

No errors with production staging tests. LGTM

**Comment by rentzso** [comment]
LGTM

**Pull request merged**